docs: helix configuration for next-ls development (#15)
elixir-tools/elixir-tools.dev: _pages/docs/next-ls/editors.md

command = "path/to/next-ls"
args = ["--stdio=true"]
```

> ### Development

Reviewer: ```suggestion
### TCP
```

Author: I named this heading `Development` in order to maintain cohesion with the heading used in [the README.md file in the next-ls repo](https://github.com/elixir-tools/next-ls?tab=readme-ov-file#development). I understand how `TCP` can make more sense, but aren't you afraid some users will fail to make the connection between `Development` and `TCP` if `Development` isn't mentioned here? Maybe there's an other way than the heading the make the connection explicit.

Reviewer: The perspective of this section of the docs is on editor configuration, not contributing. The word development doesn't necessarily imply developing Next LS. It could mean developing your application code.

> If you are running next-ls from the source code, you'll need to communicate with the server over TCP. [It is recommended to use netcat as a proxy.](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers)
> Assuming the next-ls server runs on port 9000, the LSP configuration should look as follows:

Reviewer: ```suggestion
Helix supports connecting via TCP using `netcat`. https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers
```

```toml
[[language]]
name = "elixir"
scope = "source.elixir"
language-server = { command = "nc", args = ["127.0.0.1", "9000"] }
```
If you are using the latest git version of helix use this:

```toml
[language-server.nextls]
command = "nc"
args = ["127.0.0.1", "9000"]
```

> ### Enabling completions

Reviewer: ```suggestion
### Initialization Options
```

> Completions need to be enabled explicitely in the Language Server configuration by passing intialization options under the `config` key.

Reviewer: ```suggestion
Initialization options are configured with the `config` key.
```
